huaweicloud/resource_huaweicloud_rds_configuration_v3.go

```diff
@@ -56,6 +57,12 @@ func ResourceRdsConfigurationV3() *schema.Resource {
 						"type": {
 							Type:     schema.TypeString,
 							Required: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
seems this can be wrapped as a common diff suppress function. at least we need to add this to db.type in rds_instance as well.

Reply:
This feature has been moved to the common package.